be/src/exec/vectorized/hash_join_node.cpp

```diff
@@ -183,8 +183,13 @@ void HashJoinNode::_init_hash_table_param(HashTableParam* param) {
     }
     param->predicate_slots = predicate_slots;
 
-    for (auto i = 0; i < _probe_expr_ctxs.size(); i++) {
-        param->join_keys.emplace_back(JoinKeyDesc{_probe_expr_ctxs[i]->root()->type().type, _is_null_safes[i]});
+    for (auto i = 0; i < _build_expr_ctxs.size(); i++) {
```

Comment:
originally, here is _probe_expr_ctxs, now changed to _build_expr_ctxs, is it a mistake?

Reply:
The data types of build and probe are the same, but now I need Expr in Build, so I use BuildExpr instead